tests/e2e/common.go

```diff
@@ -67,14 +67,22 @@ var (
 	defaultDeployFluentbit = false
 )
 
-func DescribeTierN(tier uint) func(string, func()) bool {
-	return func(name string, body func()) bool {
-		return Describe(fmt.Sprintf("[Tier %d] %s", tier, name), body)
-	}
+type OSMDescribeInfo struct {
+	// tier represents the priority of the test. Lower value indicates higher priority.
+	tier int
```

Comment:
Since you say `tier represents the priority` - perhaps this could be named `priority`?

Reply:
I think I came up with "tier" first to help indicate that they're a hierarchy, so e.g. all the tests in tier 1 also run as a part of tier 2. Maybe we could split the difference and call it something like "priority tier?"